data/Deepakktripathy.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name" : "Deepak Tripathy",
+  "displayStatsPublic" : true,
```

Comment:
Currently, you don't need to add the `"displayStatsPublic" : true,` code. you can delete it.
Check the updated doc https://linkfree.eddiehub.io/docs/quickstart